src/mjolnir/CMakeLists.txt

```diff
@@ -81,4 +81,5 @@ valhalla_module(NAME mjolnir
     Lua::Lua
     Threads::Threads
     ZLIB::ZLIB
-    robin_hood::robin_hood)
+    robin_hood::robin_hood
+    GEOS:GEOS)
```

Comment:
should this be?
GEOS::GEOS

Reply:
very probably :eagle: :eye:

Reply:
Yeah `::` is the right format for [alias targets in cmak](https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#alias-targets)